spec/rubocop/config_spec.rb

```diff
@@ -592,7 +592,7 @@ def cop_enabled(cop_class)
       end
 
       before do
-        allow(File).to receive(:file?).with('.ruby-version')
+        allow(File).to receive(:file?).and_call_original
```

Comment:
this test is failed in `rake parallel:ascii_spec`.
RSpec matcher is lazy loaded, and it calls `File.file?` while lazy loading.
So, I changed it that calls original method.